symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/OboEnabled.java

```diff
@@ -0,0 +1,12 @@
+package com.symphony.bdk.core.service;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
Javadoc missing + add link to https://developers.symphony.com/restapi/docs/get-started-with-obo